packages/node-name-resolver/src/NodeNameResolver.php

```diff
@@ -97,7 +97,11 @@ public function isNames(Node $node, array $names): bool
     public function isName($node, string $name): bool
     {
         if ($node instanceof MethodCall) {
-            return false;
+            $message = sprintf(
+                'Name called on "%s" is not possible. Use $this->getName($node->name) instead',
```

Comment:
```suggestion
                'Name called on "%s" is not possible. Use $this->isName($node->name) instead',
```

To avoid same PR in the future